pkg/master/master_test.go

```diff
@@ -813,7 +809,7 @@ func testInstallThirdPartyAPIDeleteVersion(t *testing.T, version string) {
 	assert.Equal(http.StatusNotFound, resp.StatusCode)
 
 	expectedDeletedKey := etcdtest.AddPrefix("ThirdPartyResourceData/company.com/foos/default/test")
-	_, err = client.Get(expectedDeletedKey, false, false)
+	_, err = etcdserver.Client.Get(expectedDeletedKey, false, false)
```

Comment:
can't you switch this to storage interface as well here?  You're not actually parsing the objects only checking existence.